packages/backend/src/graphql/resolvers/tenant.test.ts

```diff
@@ -85,7 +86,7 @@ describe('Tenant Resolvers', (): void => {
   })
 
   afterEach(async (): Promise<void> => {
-    await truncateTables(appContainer.knex, true)
```

Comment:
This is an example of the problem that lead to changing the `truncateTables` interface.